migrations/postgresql/20160909174741_create_highlight_post_view/up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE stuff (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR NOT NULL,
+  important BOOLEAN NOT NULL DEFAULT 'f'
+);
+
+CREATE VIEW important_stuff AS
```

Comment:
Good question. I think it'd be fine to leave it in but currently it doesn't give us anything, so I'll remove it. If we were to support views later, we can just re-add these migrations.